src/client.rs

```diff
@@ -612,6 +612,22 @@ impl Client {
         self.inner.shutdown.executed.store(true, Ordering::SeqCst);
     }
 
+    /// Add connections to the connection pool up to `min_pool_size`.  This is normally not needed -
+    /// the connection pool will be filled in the background, and new connections created as needed
+    /// up to `max_pool_size`.  However, it can sometimes be preferable to pay the (larger) cost of
+    /// creating new connections up-front so that individual operations execute as quickly as
+    /// possible.
+    ///
+    /// Note that topology changes require rebuilding the connection pool, so this method cannot
+    /// guarantee that the pool will always be filled for the lifetime of the `Client`.
+    pub async fn warm_connection_pool(&self) {
```

Comment:
I'm open to naming suggestions here.  "warm" seems a little vague but I couldn't think of a better one that wasn't super verbose.

Reply:
yeah this seems fine to me. can you note in the comment for this method that nothing will happen if `min_pool_size` is 0?